website/docs/r/k8s.html.md

```diff
@@ -62,6 +62,7 @@ The following arguments are supported:
     * `storage` - Storage per worker node (in GiB).
     * `storage_type` - Storage type (one of storage, storage_high, storage_insane).
     * `rocket_storage` - Rocket storage per worker node (in GiB).
+    * `taints` - List of taints to be applied to the nodes of this pool.
```

Comment:
Linking to https://my.gridscale.io/product-documentation/cloud-computing/products/paas/kubernetes/node-pools/introduction/#support-for-taints-and-labels will help